pkg/plugin/plugin.go

```diff
@@ -18,9 +18,49 @@ const (
 	MethodSendNotification = "SendNotification"
 )
 
+// FormElement describe a form element required for the channel config
+type FormElement struct {
+	// Element name
+	Name string `json:"name"`
+	// Element type: string = text, bool = checkbox, text = textarea, option = select, options = select[multiple], secret = password
+	Type string `json:"type"`
+	// Element label map. Locale in the standard format (language_REGION) as key and corresponding label as value.
+	//
+	// e.g. {"en_US": "Save", "de_DE": "Speichern"}
+	Label map[string]string `json:"label"`
+	// Element placeholder
+	Placeholder string `json:"placeholder,omitempty"`
+	// Element title
+	Help map[string]string `json:"help,omitempty"`
+	// Element default value: bool for checkbox, string for other elements
+	Default any `json:"default,omitempty"`
+	// Set true if element is required, omit otherwise
+	Required bool `json:"required,omitempty"`
+	// Select element options map:  key => value. Only required for type option and options
+	Options map[string]string `json:"options,omitempty"`
```

Comment:
In general, these comments should be more detailed than just repeating the type and field names.

Also, I think this would would look nicer and make the connection between field and comment more clear, if these were separated by blank lines.